content/en/docs/setup/production-environment/dockershim.md

```diff
@@ -0,0 +1,170 @@
+---
+title: Dockershim Deprecation FAQ
+content_type: concept
+aliases: [ '/dockershim']
+weight: 40
+---
+
+This document goes over some frequently asked questions and their answers. For
+more detail on the deprecation of Docker as a container runtime for Kubernetes
+kubelets, and what that means, check out the blog post
+[Don't Panic: Kubernetes and Docker](/blog/2020/12/02/dont-panic-kubernetes-and-docker/)
+
+### Can I still use Docker in Kubernetes 1.20?
+
+Yes, the only thing changing in 1.20 is a single warning log printed at [kubelet]
+startup if using Docker as the runtime.
+
+[kubelet]: /docs/reference/command-line-tools-reference/kubelet/
+
+### When will dockershim be removed?
+
+Given the impact of this change, we are using an extended deprecation timeline.
+It will not be removed before Kubernetes 1.22, meaning the earliest release without
+dockershim would be 1.23 in late 2021, approximately one year from now. We will
```

Comment:
I would remove the "approximately one year from now" because this won't have a date stamp on it.

Reply:
Agreed, especially as we're discussing changes to release cadence.